source/StepBro.Core/Parser/StepBroListener.ExpressionMethodCall.cs

```diff
@@ -70,7 +70,13 @@ public override void EnterArguments([NotNull] SBP.ArgumentsContext context)
 
         public override void ExitArguments([NotNull] SBP.ArgumentsContext context)
         {
-            m_arguments.Push(m_expressionData.PopStackLevel());
+            List<SBExpressionData> stackData = m_expressionData.PopStackLevel().ToList();
```

Comment:
So, what's the case where this is needed?

Reply:
An alternative of course is to do something like:
```
Stack<SBExpressionData> dataToHandle = m_expressionData.PopStackLevel();
Stack<SBExpressionData> firstRun = new Stack<SBExpressionData>();
Stack<SBExpressionData> finalData = new Stack<SBExpressionData>();
foreach(var data in dataToHandle)
{
    firstRun.Push(ResolveForGetOperation(dataToHandle.Pop()));
}
foreach(var data in firstRun)
{
    finalData.Push(data);
}
m_arguments.Push(finalData);
```
Or similar, but that seems like more hassle than the above